### PR TITLE
Fixes plone/ansible-playbook#138

### PR DESCRIPTION
**Pull request by fulv**

### CODE REVIEW

**Comment by fulv** [comment]
Fixes plone/ansible-playbook#138

File: HISTORY.txt [hunk 1]
```diff
@@ -2,7 +2,11 @@
 
 - When copying buildout extra directory, preserve file modes.
   [smcmahon]
-- Fix incorrect indentation with plone_zope_conf_additional
+
+- Fix incorrect indentation with plone_zope_conf_additional.
+  [fulv]
+
+- Disable plone_client_tcpcheck for py3.
```
Comment (stevepiercy) [inline HISTORY.txt:9]:
```suggestion
- Disable plone_client_tcpcheck for py3.
  See discussion in https://github.com/plone/ansible-playbook/issues/138
```

**Comment by tkimnguyen** [comment]
Does this PR still need merging?

**Comment by stevepiercy** [comment]
Seeing as it is not yet merged, I would say "yes". 😜 

I already approved it. Does it still need further review from @smcmahon, or can @fulv go ahead and merge it?

**Comment by tkimnguyen** [comment]
There! Now this can be blamed on me 😂

**Pull request merged**